Readme (#14)
Victoria-Moiseienko/internet-shop: README.md

##internet-shop
Internet-shop with basic functions. 

> According to N-tier architecture project contain several layers:

Reviewer: ```suggestion
In accordance with the N-tier architecture principles, this project is comprised of several layers:
```

Author: Done

* DB
* DAO
* Services
* Controllers
* Views
##main functions

> ###for Admin: 
> * Manage products
> * Manage users
> * View and delete orders
> ###for User:
> * Registration
> * Login/Logout
> * Buy products
> * View Shopping Cart
> * Create orders
> * View own orders and its details

Reviewer: ```suggestion
###for the Admin: 
* Managing products
* Managing users
* Viewing and deleting orders
###for the User:
* Registration
* Login/Logout
* Buying products
* Viewing the shopping cart
* Completing orders
* Viewing own orders and their details
```

Author: Done

##technologies 

> Java, Servlets, RBAC Filters, Tomcat, JSP, JSTL, JDBC, MySQL

Reviewer: I think you could add the version of Java you used (e.g. Java 8, Java 11) and format this list as bullet points, e.g.
- Java 8
- Servlets

Author: Done

> ##to run project
> 1. Set up MySql DB and Workbench
> 2. Run the sql-code from init_db.sql - placed in 'resources' package
> 3. Use your properties in ConnectionUtil class
> 4. Configure Tomcat

Reviewer: ```suggestion
##to run the project
1. Set up MySql DB and Workbench on your machine
2. Run the SQL-code from the _init_db.sql_ file located in the _resources_ package
3. Insert your MySQL Workbench login and password in the properties in the ConnectionUtil class
4. Configure Tomcat
```
I'd also mention the need to 'inject data' when the website is up and running and how one can try the website out as a user and an admin (e.g. you could say which login and password you've set for the admin)

Author: Done

> ##internet-shop

Reviewer 2: ```suggestion
## internet-shop
```

Internet-shop with basic functions.
 
In accordance with the N-tier architecture principles, this project is comprised of several layers:
* DB
* DAO
* Services
* Controllers

> * Views
> ##main functions
> ###for the Admin: 
> (login: admin, password: admin)
> * Managing products
> * Managing users
> * Viewing and deleting orders

Reviewer 2: ```suggestion
* Views

## main functions

### for the Admin: 
(login: admin, password: admin)
* Managing products
* Managing users
* Viewing and deleting orders
```
зараз це виглядає так:
![image](https://user-images.githubusercontent.com/36365171/96433477-e1606d80-120d-11eb-91f9-7aa2ef3b2f60.png)

Author: Done
